Documentation/ExtensionArchitecture/HowTo/Localization/Php.rst

```diff
@@ -18,6 +18,25 @@ Which method of localization to use depends on the current context:
 Localization in plain PHP
 =========================
 
+The :php:`$GLOBALS['LANG']` can be used to access the language service as
```

Comment:
We should remove this section altogether: The way to go is via LanguageServiceFactory. Encouraging users to use the global variable is not good. Also, LANG may be deprecated in v12 (as I followed the discussions about it).

Reply:
I would not remove it but shorten it and point ppl to the according chapters in Extension Development

Reply:
Why? The LANG is prominently at the beginning? It should not be used. Period. And there is always a way to use the LanguageServiceFactory with one of the create methods. 

I am okay with it, when moving a shortened part into a big "Warning" admonition not to use it.

Reply:
Because ppl are still used to using it I would like to guide them from here to the chapter of how get your Language Service from the Factory?

Reply:
Maybe also use another headline

Reply:
Yes, that's why then using a big warning. I would avoid that people come to this page, read the first sentences and use global LANG which is phasing-out and therefore should not be used anymore (at least for new code). And perhaps not at the beginning of the chapter.